chandra_aca/aca_image.py

```diff
@@ -828,3 +832,82 @@ def get_psf_image(
 
         out = ACAImage(psf, row0=row0, col0=col0) if aca_image else (psf, row0, col0)
         return out
+
+
+@retry.retry(exceptions=requests.exceptions.RequestException, delay=5, tries=3)
+def get_aca_image_table(
+    start: CxoTimeLike, stop: CxoTimeLike, bgsub=True, source="maude", **maude_kwargs
+):
+    """
+    Get ACA images and ancillary data from either the mica or maude data sources.
+
+    Parameters
+    ----------
+    start : CxoTimeLike
+        start time
+    stop : CxoTimeLike
+        stop time (CXC sec)
+    bgsub : bool
+        include background subtracted images in output table
+    source : str
+        'maude' or 'mica'
+    maude_kwargs
+        additional kwargs for maude data source
+
+    Returns
+    -------
+    Table of ACA images and ancillary data. If bgsub is True then the table
+    will include columns for the background subtracted image ('BGSUB'), the dark current
+    image ('DARK'), and the CCD temperature ('T_CCD').
+    """
+    import mica.archive.aca_dark
+    import mica.archive.aca_l0
+
+    import chandra_aca.dark_subtract
+    import chandra_aca.maude_decom
+
+    # Get aca images over the time range
+    if source == "maude":
+        imgs_table = chandra_aca.maude_decom.get_aca_images(start, stop, **maude_kwargs)
+        t_ccds = chandra_aca.dark_subtract.get_tccd_data(
+            imgs_table["TIME"], source=source, **maude_kwargs
+        )
+    elif source == "mica":
+        imgs_table = mica.archive.aca_l0.get_aca_images(start, stop)
+        t_ccds = chandra_aca.dark_subtract.get_tccd_data(
+            imgs_table["TIME"], source=source
+        )
+    else:
+        raise ValueError(f"source must be 'maude' or 'mica', not {source}")
+
+    # Do some unmasking until the maude_decom and aca_l0 interfaces are updated with
+    # fewer masked columns
+    imgs_table_unmasked = Table()
+    for col in imgs_table.colnames:
+        if col in ["TIME", "INTEG", "IMGROW0_8X8", "IMGCOL0_8X8"]:
+            imgs_table_unmasked[col] = imgs_table[col].data.data
+        else:
+            imgs_table_unmasked[col] = imgs_table[col]
+    imgs_table = imgs_table_unmasked
+
+    # Get background subtracted values if bgsub is True
+    if bgsub:
+        dark_data = mica.archive.aca_dark.get_dark_cal_props(
+            imgs_table["TIME"].min(),
+            select="nearest",
+            include_image=True,
+            aca_image=False,
+        )
+        img_dark = dark_data["image"]
+        tccd_dark = dark_data["ccd_temp"]
+        imgs_dark = chandra_aca.dark_subtract.get_dark_current_imgs(
+            imgs_table, img_dark, tccd_dark, t_ccds
+        )
+        imgs_bgsub = imgs_table["IMG"] - imgs_dark
+        imgs_bgsub.clip(0, None)
+
+        imgs_table["BGSUB"] = imgs_bgsub
+        imgs_table["DARK"] = imgs_dark
+        imgs_table["T_CCD"] = t_ccds
```

Comment:
Make it more clear that this column is a smoothed version of what comes in telemetry.
```suggestion
        imgs_table["T_CCD_SMOOTH"] = t_ccds
```